src/test/kotlin/org/rustSlowTests/console/RsEvcxrTest.kt

```diff
@@ -0,0 +1,71 @@
+/*
+ * Use of this source code is governed by the MIT license that can be
+ * found in the LICENSE file.
+ */
+
+package org.rustSlowTests.console
+
+import org.rust.cargo.RsWithToolchainTestBase
+import org.rust.cargo.project.settings.toolchain
+import org.rust.cargo.toolchain.Cargo
+import java.io.PrintWriter
+import java.util.*
+
+class RsEvcxrTest : RsWithToolchainTestBase() {
+
+    fun testProcessOutputFormat() {
```

Comment:
```suggestion
    fun `test process output format`() {
```
We use names with spaces enclosed in backticks for tests